### ISSUE TITLE
On-line thinning

### ISSUE THREAD

**Issue by patwa67** [TuringLang/Turing.jl#1607]
Is there some way to store thinned MCMC chains without doing post sampling thinning? I have a huge number of parameters that I run with AdvancedMH and only want to save lets say every 10th sample in order not to run into storage limitations.

**Comment by devmotion** [comment]
Yes, you can specify the keyword argument `thinning`. All common keyword arguments of AbstractMCMC should work since the general non-sampler specific part of the sampling is implemented in AbstractMCMC: https://turinglang.github.io/AbstractMCMC.jl/dev/api/#Common-keyword-arguments If something doesn't work, it's probably a bug.

**Comment by ithotuwanted2dance** [comment]
Hey Turing team.  I think it would be very helpful if this fact (the ability to use any of the keyword commands from AbstractMCMC) was made explicit in the Turing documentation.

**Comment by cpfiffer** [comment]
We agree -- we'd love to have someone add this to the documentation!
